beta-src/src/components/svgr-components/WDGamePhaseIcon.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from "react";
+import { Box, Button } from "@mui/material";
+import Season from "../../enums/Season";
+
+interface GamePhaseIconProps {
+  disabled?: boolean;
+  iconState: Season;
```

Comment:
I don't know that I understand the relationship between iconState and seasons. Naming seems off here.